src/components/InCallSupportMessage.jsx

```diff
@@ -0,0 +1,47 @@
+import Anchor from "../components/Anchor";
+import BodyText from "../components/text/BodyText";
+import styled from "styled-components";
+import theme from "../theme";
+
+const InCallSupportMessage = () => (
+  <FlexRow>
+    <HelpText>
+      Having trouble connecting?{" "}
+      <Anchor
+        href="https://help.daily.co/en/articles/2303117-top-troubleshooting-5-tips-that-solve-99-of-issues"
+        color={theme.colors.orange}
+      >
+        Try these fast tips
+      </Anchor>
+      , or{" "}
+      <Anchor
+        href="https://www.daily.co/contact/support?utm_source=webinar"
+        color={theme.colors.orange}
+      >
+        contact our support
+      </Anchor>{" "}
+      via chat or email.
+    </HelpText>
+    <Flex1>_</Flex1>
```

Comment:
What's this doing here?

Reply:
Literally nothing. 😎 Assuming it made sense in a previous context. Thanks for catching :)